composer.json

```diff
@@ -88,6 +88,7 @@
     },
     "autoload-dev": {
         "psr-4": {
+            "TestApp\\": "tests/Application/src/",
```

Comment:
What's the reasoning behind a new namespace?

Reply:
I've created a `src` inside the test app, as I wanted to skip putting it directly under the `tests/Application`. The namespace will look `CurrentNamespace/src/FilterBuilder` so I decided to create a shorter version, as it doesn't impact the project itself.

Reply:
so, maybe, to have it consistent with other namespaces name it `Setono\\SyliusMeilisearchPlugin\\TestApp\\`? I know it's long :dancer: but who cares when we have autocomplete 🚀